code-challenges.test.js

```diff
@@ -18,40 +18,108 @@
 
 // a) Create a test with an expect statement using the variable provided.
 
-var colors1 = ["purple", "blue", "green", "yellow", "pink"]
-// Expected output example (can be a different order): ["yellow", "blue", "pink", "green"]
+
+
+
+describe("arrayShuffle", () => {
+    var colors1 = ["purple", "blue", "green", "yellow", "pink"]
+    // Expected output example (can be a different order): ["yellow", "blue", "pink", "green"]
 var colors2 = ["chartreuse", "indigo", "periwinkle", "ochre", "aquamarine", "saffron"]
 // Expected output example (can be a different order): ["saffron", "aquamarine", "periwinkle", "indigo", "ochre"]
 
+    it("takes in an array, removes the first item from the array and shuffles the remaining content", () => {
+      expect(arrayShuffle(colors1)).not.toEqual(["purple", "blue", "green", "yellow", "pink"]) //asks test to check if the output is not equal to the original array meaning it was mutated and shuffled.
+      expect(arrayShuffle(colors2)).not.toEqual(["chartreuse", "indigo", "periwinkle", "ochre", "aquamarine", "saffron"])
```

Comment:
Haha! okay this is good, but this can also be accomplished by deleting all of the values or not returning anything at all. Still I like the move!